.github/actions/azure-kubernetes-aks-single-region-cleanup/action.yml

```diff
@@ -0,0 +1,53 @@
+---
+name: Delete Azure AKS Single Region Clusters
+
+description: |
+    This GitHub Action automates the deletion of azure/kubernetes/aks-single-region reference architecture clusters using a shell script.
+
+inputs:
+    tf-bucket:
+        description: Bucket containing the clusters states
+        required: true
+
+    tf-bucket-region:
+        description: Region of the bucket containing the resources states, if not set, will fallback on AWS_REGION
+
+    tf-bucket-key-prefix:
+        description: Key prefix of the bucket containing the resources states. It must contain a / at the end e.g 'my-prefix/'.
+        default: ''
+
+    max-age-hours-cluster:
+        description: Maximum age of clusters in hours
+        default: '12'
+
+    target:
+        description: Specify an ID to destroy specific resources or "all" to destroy all resources
+        default: all
+
+    resource-group-name:
+        description: Name of the resource group
+        required: true
+        default: camunda-rg
```

Comment:
maybe add to the description that this is only used in case of explicit retry set.
Also not required to be passed in.